smac/utils/validate.py

```diff
@@ -84,35 +81,35 @@ def __init__(self, scenario: Scenario, trajectory: list, output: str,
             num_run = np.random.randint(MAXINT)
             self.rng = np.random.RandomState(seed=num_run)
 
-        self.rh = RunHistory(average_cost)  # update this rh with validation-runs
-
-    def _save_results(self, rh:RunHistory, filename:str):
+    def _save_results(self, rh:RunHistory, output):
         """ Helper to save results to file """
-        if self.output == "":
+        if output == "":
```

Comment:
Hm, I'm not sure if output is a good name for this variable - to me this reads like that this is either the output, or whether to output something, but not where to output something.